package.json

```diff
@@ -114,7 +114,7 @@
   },
   "packageManager": "npm@8.19.4",
   "engines": {
-    "node": ">=16 || 14 >=14.18"
+    "node": ">=18"
```

Comment:
Do we not have a way around this? Can't we pin the version of `glob` to one that supports the previous version?

I just saw that v10.4.4 was released and supports `"14 >=14.21 || 16 >=16.20 || 18 || 20 || >=22"`. This is again higher than what we had before, but maybe better than dropping 14 and 16?

Although all of this is starting get pretty annoying and makes me want to rip off the bandaid and drop support so that we're done with this... but also I think that's wrong, so I'm trying really hard to resist... I'm not sure what to do yet.

Reply:
> I just saw that v10.4.4 was released and supports `"14 >=14.21 || 16 >=16.20 || 18 || 20 || >=22"`. This is again higher than what we had before, but maybe better than dropping 14 and 16?
 
Yeah. I guess we just match that.

> Although all of this is starting get pretty annoying and makes me want to rip off the bandaid and drop support so that we're done with this... but also I think that's wrong, so I'm trying really hard to resist... I'm not sure what to do yet.

I wouldn't mind, but in a patch? Ruining everyone's resolution? I'm rather unhappy about it. And it wasn't even needed. Yes, `??=` is nice, but he needed it once. And it's just syntax sugar. And just in the cli that very few even use in the first place.

Reply:
Either way, there're some other nice changes to be had here, I'll try to break them out.